utils/Azure/acrTools.ts

```diff
@@ -5,9 +5,10 @@ import * as vscode from "vscode";
 import { NULL_GUID } from "../../constants";
 import { AzureImageNode, AzureRepositoryNode } from '../../explorer/models/AzureRegistryNodes';
 import { AzureAccount, AzureSession } from "../../typings/azure-account.api";
-import { AzureImage } from "../Azure/models/image";
-import { Repository } from "../Azure/models/Repository";
 import { AzureUtilityManager } from '../azureUtilityManager';
+import { AzureImage } from "./models/image";
+import { Repository } from "./models/repository";
+const teleCmdId: string = 'vscode-docker.deleteAzureImage';
```

Comment:
I guess it's not used here.

Reply:
Removed teleCmdId, but the other two are used